modules/deploy/aws/output.tf

```diff
@@ -5,7 +5,7 @@
 #
 
 output "public_ip" {
-  value = aws_instance.cml_controller.public_ip
+  value = aws_eip.server_eip.public_ip
```

Comment:
workaround for 10m timeout error ( ignore_errors=true does not work and we also do not need this), since we only want public ipv4 under which will be the cml2 accessible

Reply:
this should not affect them

Reply:
On the time error:  it's perfectly OK to have a timeout error after 10 minutes when the state can not be retrieved until then.  As I was explaining to @ouhlar , the `ignore_errors=true` will suppress errors when retrieving the state from the controller while the controller is not there, yet.  If that would not happen, then TF will immediately show an error because it will get an "connection refused" from the (non-existent) controller.  So, if a timeout occurs after 10min, that means that the controller is NOT ready after 10min.  IOW, if the readiness state times out, then the controller is very likely NOT ready and something went wrong.